proto/chart/v1/send.proto

```diff
@@ -0,0 +1,25 @@
+syntax = "proto3";
+option go_package = "chart/send/v1;stream_charts";
+
+package chart.v1;
+
+import "google/protobuf/timestamp.proto";
+
+message StreamChartsAndDimensions {
+
+  // claim_id, node_id pair is used to identify the Node Instance
+  string claim_id = 1;
+  string node_id = 2;
+  // used to define the position of the Cloud
+  int64 seq_id = 3;
```

Comment:
Some more comments on seq_id vs batch_id would be useful for someone not really familiar with the new architecture. Something in the line of:
```suggestion
  // Last verified sequence sent by the agent
  int64 seq_id = 3;
  ..
  // Incremented every time the cloud sends a new StreamChartsAndDimensions command
  int64 batch_id = 5;
```

Reply:
thanks man I'll add them together with other adjustments in a commit.